### ISSUE TITLE
Does this project support Adaptive Bitrate Streaming?

### ISSUE THREAD

**Issue by wsw2435** [pedroSG94/vlc-example-streamplayer#30]
I have set up the "Stream Group Name" of Transcoder on the Wowza Streaming Engine server to provide the SMIL files to playback client. Does this VLC player automatically apply the "auto changing resolution and fps"? Thank you.

**Comment by pedroSG94** [comment]
No. This is only a VLC wrapper and I think that VLC doesnt support it
